mainnet.toml

```diff
@@ -0,0 +1 @@
+new_selection_algo_height = 375300
```

Comment:
Can you add a new line at the of the file? https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
Done